docs/source/quickinstall.rst

```diff
@@ -58,6 +58,49 @@ are no longer supported.
 .. _Personal Package Archive: https://launchpad.net/~paulromano/+archive/staging
 .. _APT package manager: https://help.ubuntu.com/community/AptGet/Howto
 
+-------------------------------------------
+Installing with Docker on Linux/Mac/Windows
+-------------------------------------------
+
+OpenMC can be easily deployed using `Docker <https://www.docker.com/>`_ on any
```

Comment:
Sure, this sounds fine to me. I can simplify the quick install and move this section to the developer's guide. I just thought it might make sense to include this in the installation instructions since I would argue that this may now be the simplest way for new users -- and primarily students -- to painlessly "install" OpenMC on whatever platform they may be using without running into dependency hell. But of course Docker has its own learning curve, and thus it might be of most interest to "power users" and developers.

Reply:
Btw, if you think it would be a good idea to refer users to Dockerhub in the quick install guide, what do you think about creating an `openmc-dev` Dockerhub user with an `openmc` repo?

Reply:
Good idea. I just created an `openmc` organization and added you as an owner.

Reply:
My latest commit simplified the quick installation by referring user's to `run` the image from DockerHub. I moved the more detailed description of how to build a Docker image to a new sub-section at the end of the Dev Guide.